apps/src/JSInterpreter.js

```diff
@@ -45,6 +45,7 @@ var JSInterpreter = module.exports = function (options) {
   this.paused = false;
   this.yieldExecution = false;
   this.startedHandlingEvents = false;
+  this.executionError = null;
```

Comment:
For now, each `JSInterpreter` can only be parsed and executed once. But I agree, if we change that, we will need to be sure to reset this.